packages/es-components-via-theme/index.js

```diff
@@ -493,6 +496,7 @@ const theme = {
   font: {
     baseFontSize: '18px',
     baseLineHeight: 1.428,
+    baseFontFace: 'Source Sans Pro',
```

Comment:
In other places in es-components font- face is `'Source Sans Pro', 'Segoe UI', Segoe, Calibri, Tahoma, sans-serif;`  Do we need to do that here as well?  (I don't know the answer, but maybe Steve or Ken will know).

Reply:
Yeah, and the ticket only calls for using that font in text inputs/areas. I could move it to `theme.formInputs.baseFontFace` or something.

Reply:
Yeah, those are fallback font faces that should be installed on most Mac/Windows computers by default. If they haven't included the reference to our preferred `Source Sans Pro` then at least there's some consistency in what is used instead.

Reply:
I would recommend that we use the whole fonts list, but I think this is appropriate to stay on the `font` object, and I think it's a valuable addition!